### ISSUE TITLE
TOPICS: Users aren't taken to the active phase tab on mobile

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1375]
@ilmartyrk @kevincrepin 

I'm not sure if this is a bug or deliberate, but something to consider: 

**Insights from Round 2 ideation participants tests:** 

It seems that on mobile users aren't getting taken to the tab for the phase which is currently "active", when following a share link to it. In the test, I ran the scenario as follows, which seems quite realistic to how it would often happen. 

There were community decision-making processes going on, and for each phase of the topic, the participants were sent a message, or viewed a social media post, encouraging them to participate in the next phase of the topic. E.g.: 

- "What shall we do with funding for the library? Add your ideas here: URL"
- "Should we even keep it as a library? Join the discussion to discuss the pros and cons here: URL"
- "It was decided we'll keep it as a library. Vote now for what to spend the money on here: URL" 

The two participants on mobile, both pointed out that they would expect to be taken to the relevant tab. They were instead taken to the topic text tab each time (which they had already seen the first time). They had to try and find their way to the right tab, and while they did so, it wasn't always totally clear to them, and was helped by the messages I'd sent having clues (e.g. 'discuss the pros and cons', or 'vote') which wouldn't always be the case. They both pointed out that they would expect to be taken to the relevant part, if they were following a link from a message referencing that part.

**Suggestions**

Initially I was thinking this was just a bug, but I'm wondering if it was deliberate, as some people who follow the link at a later phase (e.g. voting) might not yet have read the topic text, and this might be important? 

Some ideas for how to solve it: 
- @ilmartyrk Is there a way for the system to detect if someone has already joined the topic. If they haven't, take them to the first tab, if they have, take them to the active phase tab? 
- As a quick fix, we could just change it so that it does take all users to the active phase tab, but hope that if they've not yet seen the earlier phases (and topic text), they click around a bit to see what's what?

**Comment by kevincrepin** [comment]
I agree if it's possible that a shared link that speaks about (participating in) a specific phase, it would be great if that link also directs to that correct phase. Considering the future possibility of each phase having its own text as well there is probably no real reason for pushing people to the info part of the topic. But if the topic is shared "as it is" then probably good to still funnel ppl first through there.

**Comment by BeccaMelhuish** [comment]
@kevincrepin OK yes makes sense - though just to check I followed correctly, this means I'll close this issue for now then, but re-awaken it once we have separate phase texts?

**Comment by kevincrepin** [comment]
I think it can stay open for @ilmartyrk. I think it can work with or without the separate phase texts, it's just that at the moment I would understand we push the user through the info tab because we want them to understand the topic before engaging with it. But with separate phase texts the phase text itself should bring that understanding.

**Comment by BeccaMelhuish** [comment]
Will put this into 'later' as is connected to the 'bigger fixes' :)
